src/components/invoiceForm.vue

```diff
@@ -64,6 +76,11 @@
         includeNote: {
           subject: '',
           content: ''
+        },
+        sellerTradeParty: {
+          id: '',
+          globalID: '',
```

Comment:
The template uses `globalid` whilst the the state uses `globalID`, with differences in capitalization. Is this intended?